data-raw/process_data_sets_meredith.R

```diff
@@ -0,0 +1,143 @@
+candy_rankings <- read_csv("data-raw/candy-power-ranking/candy-data.csv")
+# snake_case for variable names
+candy_rankings <- rename(candy_rankings, c("competitorname"="competitor_name", 
+                                          "peanutyalmondy"="peanuty_almondy", 
+                                          "crispedricewafer"="crisped_ricewafer", 
+                                          "sugarpercent"="sugar_percent", 
+                                          "pricepercent"="price_percent", 
+                                          "winpercent"="win_percent"))
+# Convert binary variables to logical variables
+cols.num <- c("chocolate", "fruity", "caramel", "peanuty_almondy", "nougat", "crisped_ricewafer",
+              "hard", "bar", "pluribus")
+candy_rankings[cols.num] <- lapply(candy_rankings[cols.num], as.logical)
+devtools::use_data(candy_rankings, overwrite = TRUE)
+
+
+# chess-transfers ----------------------------------------------------------------
+chess_transfers <- read_csv("data-raw/chess-transfers/transfers.csv")
+# snake_case for variable names
+colnames(chess_transfers) <- colnames(chess_transfers) %>%
+  tolower() %>%
+  str_replace_all("\\.", "_") %>%
+  str_replace_all(" ", "_")
+# Change transfer date to proper format
+chess_transfers$transfer_date <- as.Date(chess_transfers$transfer_date, "%m/%d/%y")
+# Convert numeric variable to character
+chess_transfers$id <- as.character(chess_transfers$id)
+devtools::use_data(chess_transfers, overwrite = TRUE)
+
+# congress-generic-ballot --------------------------------------------------------
+# generic_polllist
+generic_polllist <- 
+  read_csv("https://projects.fivethirtyeight.com/generic-ballot-data/generic_polllist.csv")
+# snake_case for variable names
+generic_polllist <- rename(generic_polllist, c("modeldate"="model_date", "startdate"="start_date",
+                                               "enddate"="end_date", "samplesize"="sample_size", 
+                                               "createddate"="created_date"))
+# Change model/start/end/created date to proper format
+generic_polllist$model_date <- as.Date(generic_polllist$model_date, "%m/%d/%Y")
+generic_polllist$start_date <- as.Date(generic_polllist$start_date, "%m/%d/%Y")
+generic_polllist$end_date <- as.Date(generic_polllist$end_date, "%m/%d/%Y")
+generic_polllist$created_date <- as.Date(generic_polllist$created_date, "%m/%d/%Y")
+generic_polllist$timestamp <- parse_date_time(generic_polllist$timestamp, "HMS dmY")
+# Convert character variables to factor
+cols.num <- c("subgroup", "pollster", "grade", "population" )
+generic_polllist[cols.num] <- lapply(generic_polllist[cols.num], as.factor)
+# Convert integer variables to character
+cols.num <- c("poll_id", "question_id")
+generic_polllist[cols.num] <- lapply(generic_polllist[cols.num], as.character)
+devtools::use_data(generic_polllist, overwrite = TRUE)
+
+# generic_topline
+generic_topline <- 
```

Comment:
Could you also create a folder `congress-generic-ballot` and download the CSV's there? I'd still `read_csv` from the online version, but make a note that we have a backup of the csv just in case the data from the web goes offline